### PR TITLE
feat: Use the new bridge signaling format.

### PR DESCRIPTION
**Pull request by jallamsetty1**
Translate the 'LastNChangedEvent', 'SelectedEndpointsChangedEvent' and 'ReceiverVideoConstraint' messages into the new 'ReceiverVideoConstraints' message that invokes the new bandwidth allocation algorithm in the bridge that is described here - https://github.com/jitsi/jitsi-videobridge/blob/master/doc/allocation.md. useNewBandwidthAllocationStrategy=true in config.js will invoke the translation in the client.

### CODE REVIEW

File: modules/qualitycontrol/ReceiveVideoController.js [hunk 1]
```diff
@@ -1,6 +1,139 @@
...
+        this._maxFrameHeight = MAX_HEIGHT_ONSTAGE;
+
+        // The endpoint IDs of the participants that are currently selected.
+        this._selectedEndpoints = [];
+
+        this._receiverVideoConstraints = {
+            constraints: {},
+            defaultConstraints: this.defaultConstraints,
+            lastN: this._lastN,
+            selectedEndpoints: this._selectedEndpoints
```
Comment (bgrozev) [inline modules/qualitycontrol/ReceiveVideoController.js:36]: nit: Also initializee `onStageEndpoints`?

File: modules/qualitycontrol/ReceiveVideoController.js [hunk 2]
```diff
@@ -80,7 +237,14 @@ export class ReceiveVideoController {
         this._maxFrameHeight = maxFrameHeight;
 
         for (const session of this._conference._getMediaSessions()) {
-            maxFrameHeight && session.setReceiverVideoConstraint(maxFrameHeight);
+            if (session.isP2P || !this._receiverVideoConstraints) {
```
Comment (bgrozev) [inline modules/qualitycontrol/ReceiveVideoController.js:240]: Is `setReceiverVideoConstraint` actually used in p2p?

Comment (jallamsetty1) [reply modules/qualitycontrol/ReceiveVideoController.js:240]: Yes, JingleSessionPC.setReceiverVideoConstraint sends out a content-modify for requesting a different resolution in p2p mode and uses the legacy format for jvb mode. https://github.com/jitsi/lib-jitsi-meet/blob/master/modules/xmpp/JingleSessionPC.js#L1311

File: modules/qualitycontrol/ReceiveVideoController.js (hunk 1 above)
Comment (jallamsetty1) [reply modules/qualitycontrol/ReceiveVideoController.js:36]: Good catch, will initialize `onStageEndpoints` as well.

**Review by bgrozev — APPROVED** [review]

**Pull request merged**